docs/workflows/best-practices-organization.md

```diff
@@ -0,0 +1,97 @@
+
+## Three stakeholder groups in a hybrid model
+
+Typically, nobody in an organization is opposed to building quality software. Rather, investing in software quality often loses out to other topics, typically adding features and fixing bugs. The shift that needs to take place is that organizations allow for a fair competition to take place between feature work and technical improvements. 
+
+Having consulted with many organizations, we have identified three stakeholder groups that need to be involved to make this shift and achieve high quality software, structurally. These are:
+- Teams
+- Engineering leads
+- Management
+
+Making the shift is not just a matter of 'the business should allow us to work on improvements instead of pushing for features'. There is a responsibility from teams and engineering leads to properly shape and quantify the work they think needs to be done so a fair assessment can be made. Also, teams need to accept the need for standards. In areas like security and legal, it is not practical or desirable for individual teams to assume full responsibility for these aspects. 
+
+The picture below depicts the hybrid model that enables this shift. It aims to strike a middle ground between a top-down model that leaves teams with very little autonomy to address problems as they see fit and the fully autonomous model where management has very little control even if that would benefit the teams.
+
+<img src="../images/hybrid-model.png" width="700" />
+
+### Who is in each group?
+
+While our terminology is pretty generic, each organization is different and some mapping will need to be done to map the different groups to existing roles in an organization.
+
+Of these groups, **teams** are typically the easiest to identify, because they exist as such. 
+
+**Management** can be harder because many organizations, especially larger ones, have multiple layers of management and not all of them are relevant in our context. Typically though, management represents the (strategic) organizational goals and has a budget responsibility. They are accountable for the outcomes of software development and as such, are concerned with topics like cost, speed and risk. While this accountability ultimately lies with a CEO, the concerns can also be represented by a product owner or project manager.
+
+**Engineering leads** are typically the hardest one to identify because it's of the three the least established and exists in various forms and to varying degrees. Engineering leads are typically involved in (higher-level) solution design, technology choices or security analyses and policies. Job titles may include tech lead, team lead or architect. Engineering leads may exist within teams and organize virtually through Communities of Practice, Guilds or Chapters. In other cases, separate engineering lead roles or teams exist, such as enterprise architects, security specialists or Centers of Excellence. 
```

Comment:
I would make a comment that we refer to these people as the architect persona elsewhere in the documentation (to avoid every page using slightly different termininology).

Reply:
Will do